syntax = "proto3";
package tinkoff.proto;

// Request
message CreateStepInDealRequest {
  string deal_id = 1;
  string description = 2;
}

// Response
message CreateStepInDealResponse {
  string deal_id = 1;
  string step_id = 2;
  uint32 step_number = 3;
  string description = 4;
  string status = 5;
}